ubuntu-advantage.1

```diff
@@ -136,6 +136,12 @@ config file to /etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy
 \fBapt_https_proxy\fP
 If set, ua will configure apt to use the specified https proxy by writing a apt
 config file to /etc/apt/apt.conf.d/90ubuntu-advantage-aptproxy
+.TP
+.B
+\fB<job_name>_timer\fP
+Sets the timer running interval for a specific job. Those intervals are checked
+every time the systemd timer runs. Run \fBua config show\fP for a list of
+configurable jobs.
```

Comment:
I wrote this based on the proxy examples - without going deep in details. Is this enough for the manpage?

Reply:
I think that's fine, but we must have a manpage entry for each job that we are running. But this should be on a separate PR